chapter-10-workload-identities/dbserver/dbserver.yaml

```diff
@@ -76,6 +81,25 @@ spec:
             readOnly: true
           - name: svids
             mountPath: /svids
+      initContainers:
```

Comment:
The init container uses a SPIFFE helper configuration file with daemon_mode = false. Once the init container runs to completion, the main container starts and SVIDs are available to it. This feels like a better technique to represent to the reader.